src/Command.ts

```diff
@@ -112,7 +112,8 @@ export default abstract class Command {
         autocomplete: this.allowedOutputs
       },
       { option: '--verbose' },
-      { option: '--debug' }
+      { option: '--debug' },
+      { option: '--interactive' }
```

Comment:
`--interactive` shouldn't be a command option. Instead, it should be a configuration setting that can be set using `cli config set`